http-server-netty/src/main/java/io/micronaut/http/server/netty/binders/NettyBodyAnnotationBinder.java

```diff
@@ -104,6 +105,10 @@ public BindingResult<T> bindFullBody(ArgumentConversionContext<T> context, HttpR
             return super.bindFullBody(context, source);
         }
         if (nhr.byteBody().expectedLength().orElse(-1) == 0) {
+            String defaultValue = context.getAnnotationMetadata().stringValue(Body.class, "defaultValue").orElse(null);
```

Comment:
```suggestion
            String defaultValue = context.getAnnotationMetadata().stringValue(Bindable.class, "defaultValue").orElse(null);
```

adjust import accordingly